### PR TITLE
WFLY-14634 Upgrade Infinispan to 12.1.1.Final

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-14634

### CODE REVIEW

**Comment by pferraro** [comment]
ProtoStream 4.4.0.Final is out and integrated.  Removing hold label.

File: pom.xml [hunk 1]
```diff
@@ -393,8 +393,8 @@
         <version.org.hibernate.validator>6.0.22.Final</version.org.hibernate.validator>
         <version.org.hornetq>2.4.7.Final</version.org.hornetq>
         <!-- n.b. Infinispan Server version used for integration testing is defined separately in testsuite/integration/clustering/pom.xml -->
-        <version.org.infinispan>11.0.10.Final</version.org.infinispan>
-        <version.org.infinispan.protostream>4.3.5.Final</version.org.infinispan.protostream>
+        <version.org.infinispan>12.1.1.Final</version.org.infinispan>
+        <version.org.infinispan.protostream>4.4.0.Beta3</version.org.infinispan.protostream>
```
Comment (bstansberry) [inline pom.xml:397]: 4.4.0.Final is released

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rhusar) [reply pom.xml:397]: Looks like that commit got lost in a rebase and force push.

**Review by rhusar — CHANGES_REQUESTED** [review]

**Review by rhusar — APPROVED** [review]
No more comments - looks great!

**Comment by pferraro** [comment]
Great. I'll squash the above commits.

File: clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java [hunk 2]
```diff
@@ -141,4 +143,8 @@
     @LogMessage(level = INFO)
     @Message(id = 32, value = "Getting remote cache named '%s'. If it does not exist a new cache will be created from configuration template named '%s'; null value uses default cache configuration on the Infinispan Server.")
     void remoteCacheCreated(String remoteCacheName, String cacheConfiguration);
+
+    @LogMessage(level = WARN)
+    @Message(id = 33, value = "Attribute '%s' configured to use deprecated value: %s; use one of the following values instead: %s")
+    void enumValueDeprecated(String attributeName, Object attributeValue, Set<?> supportedValues);
```
Comment (bstansberry) [inline clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java:149]: This is kind of a nit, but this method name kind of encourages misuse, i.e. reuse in a very different context, where any kbase article / google search result looking for the message id would find content that's not meaningful.

Also

s/configured/is configured/g
s/use deprecated/use a deprecated/g

Comment (pferraro) [reply clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/InfinispanLogger.java:149]: will fix

**Review by bstansberry — APPROVED** [review]

**Pull request merged**